.github/workflows/ci.yml

```diff
@@ -60,9 +60,10 @@ jobs:
 
       - name: Upload coverage
         uses: codecov/codecov-action@v4
+        env:
+          CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
```

Comment:
@kt3k, please ensure this token has been added ([documentation](https://docs.codecov.com/docs/adding-the-codecov-token#github-actions)).

Reply:
This is already set